packages/login/src/graphql/mutations/auth.js

```diff
@@ -13,6 +13,13 @@ export const SUBMIT_TOKEN_MUTATION = gql`
   mutation signInVerifyAuthenticationCode($phone: String!, $code: String!) {
     signInVerifyAuthenticationCode(phone: $phone, code: $code) {
       jwt
+      user {
```

Comment:
an idea, we can pass a value to this mutation query or use a default. 
If we add a new param to the user, we need to change it on UI every time a client needs it. 

Thoughts?